src/modules/shared/pages/switch-context/switch-context.component.ts

```diff
@@ -85,7 +86,8 @@ export class PageSwitchContextComponent  extends CoreComponent implements OnInit
       })
   
       if (!this.initialSelection) {
-        this.setRedirectAlertSuccess(`Switch successful: you are now logged in with your ${roleName} profile.`);
+        const message = currentOrgUnitId === organisation.organisationUnits.id ? `You are logged in as ${this.isAccessor ? 'an' : 'a'} ${roleName}.` : `Switch successful: you are now logged in with your ${roleName} profile.`
```

Comment:
I feel like the variable `organisationUnits` is kinda strange, since it's just a single unit. I had to double-check the code to see if it was an array or a single object